toolkit/docs/amd/amd.md

```diff
@@ -0,0 +1,12 @@
+# AMD Repository Configuration
+
+## Overview
+The following documentation describes how to access Azure Linux packages from the AMD RPM repository at [packages.microsoft.com](https://packages.microsoft.com/azurelinux/3.0/prod/amd/)
+
+## Instructions
+The following instructions register the amd package store with the package manager and install the amdgpu package.
+```ls
+sudo tdnf install -y azurelinux-repos-amd
+sudo tdnf repolist --refresh
+sudo tdnf install -y amdgpu
```

Comment:
to install for an older kernel, you may need to specify the version, for example: sudo tdnf install amdgpu-6.8.5.60202_2078359